<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajouter_client</class>
 <widget class="QDialog" name="ajouter_client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/sig/erp.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../interface_gestion_des_ventes-build-desktop-Qt_4_8_1_for_Desktop_-_MinGW__Qt_SDK__Debug/images/erp.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>750</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="iconnes.qrc">:/sig/images/erp.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="dernierSjr_lab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>560</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dernier séjour</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nom_le">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="CIN_lab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>440</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="prenom_le">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="cin_le">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="adresse_lab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>520</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adresse</string>
   </property>
  </widget>
  <widget class="QLabel" name="naissance_lab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_lab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N° téléphone</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="naissance_le">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>1341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="prenom_lab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>360</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sejour_le">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>560</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_le">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>480</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nom_lab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>320</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="fiche_lab">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>FICHE DE CLIENT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="adresse_le">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enregistrer_Pb">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>620</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iconnes.qrc"/>
 </resources>
 <connections/>
</ui>
